src/lib/hooks/states/project.js

```diff
@@ -1,14 +1,15 @@
 import { useQuery } from "@apollo/client";
 
-import Projects from "src/graphql/queries/projects";
+import Project from "src/graphql/queries/project";
 
-export const useProjects = () => {
-  const { data, loading, error } = useQuery(Projects, {
+export const useProject = ({ id }) => {
+  const { data, loading, error } = useQuery(Project, {
     fetchPolicy: "cache-and-network",
+    variables: { id },
   });
 
   return {
-    projects: data?.projects || [],
+    tasks: data?.project?.tasks || [],
```

Comment:
I think useProject hook should return project 
```suggestion
    tasks: data?.project || {},
```